warp10/src/main/java/io/warp10/script/StackUtils.java

```diff
@@ -212,6 +212,7 @@ public static void objectToJSON(JsonSerializer serializer, PrintWriter out, Obje
         long ts = decoder.getTimestamp();
         long location = decoder.getLocation();
         long elevation = decoder.getElevation();
+        // We do not call getBinaryValue because JSON cannot represent byte arrays
```

Comment:
May result in an invalid JSON, see this WarpScript:
```
NEWENCODER
1 1 1 1 '00' HEX-> ADDVALUE
```
Problem is that for GTSs, see line 167, we don't know that if a string represents a byte[] so it won't be straightforward to find a solution.